modules/labs-react/expandable/stories/Expandable.stories.mdx

```diff
@@ -0,0 +1,122 @@
+import {Meta} from '@storybook/addon-docs';
+import {Specifications} from '@workday/canvas-kit-docs';
+
+import {Expandable} from '@workday/canvas-kit-labs-react/expandable';
+
+import {StartIcon} from './examples/StartIcon';
+import {EndIcon} from './examples/EndIcon';
+import {Avatar} from './examples/Avatar';
+import {Depth} from './examples/Depth';
+import {RTL} from './examples/RTL';
+
+<Meta title="Labs/Expandable/React" component={Expandable} />
+
+# Canvas Kit Expandable
+
+Expandable container enables users to conditionally show and hide content.
+
+## Installation
+
+```sh
+yarn add @workday/canvas-kit-labs-react
+```
+
+## Usage
+
+### Start Icon
+
+<ExampleCodeBlock code={StartIcon} />
+
+### End Icon
+
+<ExampleCodeBlock code={EndIcon} />
+
+### With Avatar
+
+<ExampleCodeBlock code={Avatar} />
+
+### Right to Left (RTL)
+
+<ExampleCodeBlock code={RTL} />
+
+### Depth
+
+<ExampleCodeBlock code={Depth} />
+
+## Components
+
+### Expandable
+
+#### Usage
+
+`Expandable` is a container component that holds a `Expandable.Target` and a `Expandable.Content`.
+It provides a React Context model for its sub components.
+
+#### Props
+
+<!-- <ArgsTable of={Expandable} /> -->
+
+### Expandable.Target
+
+#### Usage
+
+`Expandable.Target` creates a heading and a button. The button comes with the `aria-expanded` and
+`aria-controls` attributes. When a user interacts with the button, they can show or hide the
+`Expandable.Content`. If a string is passed to the Target as a child, it will be wrapped in a Title.
+
+#### Props
+
+<!-- <ArgsTable of={Expandable.Target} /> -->
```

Comment:
I've commented out the prop tables and will update them once I've gotten some feedback on the components/API 🙂

Reply:
It's great idea. We're using fake components to get just specific props instead of all.